packages/graphql-transformers-e2e-tests/src/__tests__/NonModelAuthV2Function.e2e.test.ts

```diff
@@ -35,8 +35,8 @@ const awsS3Client = new S3({ region: REGION });
 const iamHelper = new IAMHelper(REGION);
 
 const BUILD_TIMESTAMP = moment().format('YYYYMMDDHHmmss');
-const STACK_NAME = `FunctionTransformerTestsV2-${BUILD_TIMESTAMP}`;
-const BUCKET_NAME = `appsync-function-v2-transformer-test-bucket-${BUILD_TIMESTAMP}`;
+const STACK_NAME = `NonModelAuthV2FunctionTransformerTests-${BUILD_TIMESTAMP}`;
+const BUCKET_NAME = `non-model-auth-function-v2-transformer-test-bucket-${BUILD_TIMESTAMP}`.substring(0, 63);
```

Comment:
The bucket name can be between 3 and 63 characters long, and can contain only lower-case characters, numbers, periods, and dashes.
https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html